src/app/stack/card-details/component-information/component-information.component.spec.ts

```diff
@@ -1,167 +1,167 @@
-    });
-
-    it('check if the input received in "positions" is of type MComponentHeaderColumn', () => {
-        expect(component.positions[0] instanceof MComponentHeaderColumn).toBeTruthy();
-    });
-
-    it('should expand if "isOpen" is true', () => {
-        (<MComponentInformation>component.component).isOpen = true;
-        fixture.detectChanges();
-        let parent = element.querySelector('.analytics-component-information');
-        expect(parent.querySelector('.accordion-body')).toBeDefined();
-    });
-
-    it('should be normal if "isOpen" is false', () => {
-        (<MComponentInformation>component.component).isOpen = false;
-        fixture.detectChanges();
-        let parent = element.querySelector('.analytics-component-information');
-        expect(parent.querySelector('.accordion-body')).toBeNull();
-    });
-
-    it('should have a split view if recommendation is present with component', () => {
-        (<MComponentInformation>component.component).isOpen = true;
-        (<MComponentInformation>component.component).recommendation = new MRecommendationInformation(
-            'alternate',
-            'This is the reason',
-            null,
-            null,
-            new MComponentInformation(
-                'alternate-1',
-                '',
-                '',
-                null,
-                false,
-                false,
-                false,
-                null,
-                null,
-                null,
-                null,
-                'Create work item',
-                false,
-                null,
-                false,
-                null,
-                '',
-                '',
-                null
-            ),
-            '',
-            null
-        );
-        fixture.detectChanges();
-        let parent = element.querySelector('.analytics-component-information');
-        expect(parent.querySelector('.accordion-body').children.length).toBe(2);
-    });
-
-    // TODO should improve coverage
-
-});
+// import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+// import { Http, ConnectionBackend, RequestOptions, BaseRequestOptions } from '@angular/http';
+
+// import { ComponentInformationComponent } from './component-information.component';
+
+// import { TooltipModule } from 'ngx-bootstrap';
+// import { ToastNotificationModule } from '../../toast-notification/toast-notification.module';
+// import { ComponentSnippetComponent } from './component-snippet/component-snippet.component';
+
+// import { ProgressMeterModule } from '../../utils/progress-meter/progress-meter.module';
+// import { ComponentFeedbackModule } from '../../utils/component-feedback/component-feedback.module';
+
+// import {
+//     MRecommendationInformation,
+//     MComponentInformation,
+//     MGenericStackInformation,
+//     MComponentHeaderColumn,
+//     MGithub,
+//     MOsio
+// } from '../../models/ui.model';
+
```

Comment:
Please remove all commended codes.

Reply:
Removed them, these are old test case which are no longer of use.